Source/WebCore/xml/XPathUtil.h

```diff
@@ -27,6 +28,7 @@
 #pragma once
 
 #include <wtf/Forward.h>
+#include <wtf/unicode/CharacterNames.h>
```

Comment:
Why is this include needed?

Reply:
I added it to use Unicode characters similar to other files but removing it does compile as well. So I will remove it.

Reply:
You only need this if you actually need to use one of the constants defined in that file.

Reply:
Done.